src/tailwind.js

```diff
@@ -48,6 +48,7 @@ module.exports = {
           default: "#0099cc",
           300: "#f3f8fc",
           400: "#a6c1ee",
+          600: "#0288b5",
```

Comment:
this is used only for the stimulation yeah? if thats the only use case maybe we use some other color instead of adding a new one?

Reply:
ok